src/test/ui/argument-suggestions/basic.stderr

```diff
@@ -25,8 +25,9 @@ LL | fn extra() {}
    |    ^^^^^
 help: remove the extra argument
    |
-LL |     extra();
-   |          ~~
+LL -     extra("");
```

Comment:
What do think about shrinking the replacement span in those simple cases where the diff is (- line, +line) to reduce verbosity?